### ISSUE TITLE
Thinking about placing a fuse on the 5 V input

### ISSUE THREAD

**Issue by etkaar** [skot/bitaxeGamma#17]
While the Mean Well PSUs are short-circuit safe, it may be wise to place a fuse on the 5V input as MLCCs can fail and become very hot. Some users use high-power PSUs for multiple Bitaxes, so I am unsure if that would be safe without an extra fuse.

A Littlefuse 217 Series (fast) with up to 40 A interruption rating has a voltage drop of 130 mV at 4 A:
https://cdn-reichelt.de/documents/datenblatt/C400/217_ENG_DATASHEET.pdf

I welcome any feedback from electrical engineers!

**Comment by skot** [comment]
I don't think 130mV drop is going to be good with only 5V anyways.

Either way a traditional fuse is very bad user experience in my opinion. Also the overclockers are not going to like it. I would consider a resettable, adjustable eFuse... If you know of a good one.

We currently do have the ability to do overcurrent shutdown in the TPS546 voltage regulator.

**Comment by skot** [comment]
Yes, As I mentioned above I don't think the is the right approach.
